ddtrace/internal/datadog/profiling/ddup/CMakeLists.txt

```diff
@@ -81,4 +89,37 @@ if(LIB_INSTALL_DIR)
         LIBRARY DESTINATION ${LIB_INSTALL_DIR}
         ARCHIVE DESTINATION ${LIB_INSTALL_DIR}
         RUNTIME DESTINATION ${LIB_INSTALL_DIR})
+
+    # Windows doesn't have an equivalent of RPATH. Instead, we create a symlink to dd_wrapper file from the
```

Comment:
If this works in the final wheel, I think it's fine for now, but in the long run we may need a more advanced solution for loading dynamic dependencies on Windows.

Reply:
Agreed. Once we can migrate to use PyO3, we might be able to remove this.